test/utf8/surrogatepair.js

```diff
@@ -7,4 +7,9 @@
 // For this test case to work, please save this file with UTF-8 encoding
 var y = "function () { 'চ𐌲𐌿𐍄𐌹𐍃𐌺নির্বাচিত নিবন্ধ।' ;WScript.Echo('hello'); }"
 var x = function () { 'চ𐌲𐌿𐍄𐌹𐍃𐌺নির্বাচিত নিবন্ধ।' ;WScript.Echo('hello'); }
-WScript.Echo(x.toString() === y ? "PASS" : "FAIL");
\ No newline at end of file
+
+// 2 bytes 
+var y2 = "function () { '�� kugu' ;WScript.Echo('hello'); }" 

```

Comment:
this and above looks like corrupted. If you have copy-pasted the contents, could you please move the file as is from my PR instead?